app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Admin::DashboardController < AdminController
+  before_action -> { current_page_title('Dashboard') }
+
+  def index
+    @active_subscriptions_count = Subscription.where(active: true).count
+    subscriptions = Donation.where(status: 'finished', donation_type: Donation::DONATION_TYPES[:subscription])
```

Comment:
@castrolem I think we could this in SQL instead https://apidock.com/rails/ActiveRecord/Calculations/sum

Reply:
Changed